### PR TITLE
feat(docs): add hover effect to footer socials

### LINKED ISSUE #2683: [FEAT] Add hover to socials in page
**Describe the bug**
Currently there is no hover effect for the socials icon on the docs page.  A hover over the icons would make it better recognizable to the user.

**To Reproduce**
1. Go to [https://refine.dev/docs/getting-started/overview/](https://refine.dev/docs/getting-started/overview/)
2. Go to footer section
3. Point to any socials icon

**Expected behavior**
A hover effect could be added.

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser chrome
 - Version 106

**Smartphone (please complete the following information):**
 - Used dataProvider [e.g. custom, Strapi, Airtable]

**Additional context**
If it seems relevant, can I work on it?

### PR DESCRIPTION
**Pull request by developer-diganta**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

_Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request._

Please provide enough information so that others can review your pull request:

Explain the **details** for making this change. What existing problem does the pull request solve?

I have added a hover effect to the social icons present in the footer

### Test plan (required)

Demonstrate the code is solid. If not, please add `WIP:` in its title.

### Closing issues

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).
Closes #2683 
### Self Check before Merge

Please check all items below before review.

-   [x] Corresponding issues are created/updated or not needed
-   [x] Docs are updated/provided or not needed
-   [x] Examples are updated/provided or not needed
-   [x] TypeScript definitions are updated/provided or not needed
-   [x] Tests are updated/provided or not needed
-   [x] Changesets are provided or not needed

### CODE REVIEW

**Comment by alicanerdurmaz** [comment]
Hi @developer-diganta, Thank you for helping us 🙏 

I think we should use the color of the brand owners.

github: #171515
discord: #5865F2
reddit: #FF4300
twitter: #1DA1F2
linkedin: #0077B5

**Comment by developer-diganta** [comment]
Sure! I'll make the changes.

File: documentation/src/theme/Footer/Layout/index.js [hunk 1]
```diff
@@ -183,25 +177,25 @@ export default function FooterLayout({
                             {socialLinks?.items?.map?.((socialLink, i) => (
                                 <a
                                     key={i}
-                                    className="w-[22px] h-[22px]"
+                                    className="w-[22px] h-[22px] hover:text-red-600"
```
Comment (alicanerdurmaz) [inline documentation/src/theme/Footer/Layout/index.js:180]:
```suggestion
                                    className="w-[22px] h-[22px]"
```

We don't have text link right now and probably red will not fit our design system.
We can delete this hover style right now.

**Review by alicanerdurmaz — APPROVED** [review]

**Comment by alicanerdurmaz** [comment]
@developer-diganta Thanks for helping us 🚀

**Review by omeraplak — APPROVED** [review]

**Pull request merged**